src/components/FriendsItemsBox.vue

```diff
@@ -113,16 +117,17 @@
                 });
             },
             addItem() {
+                var self = this;
                 this.selectedFriends.forEach(function (item) {
```

Comment:
you had to use the `self` trick because of this `function`

you can simply use an arrow function and `this` would work just fine. plz see how JS functions use and get their context (`this`)